// faker/lib/locales.js
exports['az'] = require('./locales/az/index');
exports['cz'] = require('./locales/cz/index');
exports['de'] = require('./locales/de/index');
exports['de_AT'] = require('./locales/de_AT/index');
exports['de_CH'] = require('./locales/de_CH/index');
exports['en'] = require('./locales/en/index');
exports['en_AU'] = require('./locales/en_AU/index');
exports['en_BORK'] = require('./locales/en_BORK/index');
exports['en_CA'] = require('./locales/en_CA/index');
exports['en_GB'] = require('./locales/en_GB/index');
exports['en_IE'] = require('./locales/en_IE/index');
exports['en_IND'] = require('./locales/en_IND/index');
exports['en_US'] = require('./locales/en_US/index');
exports['en_au_ocker'] = require('./locales/en_au_ocker/index');
exports['es'] = require('./locales/es/index');
exports['es_MX'] = require('./locales/es_MX/index');
exports['fa'] = require('./locales/fa/index');
exports['fr'] = require('./locales/fr/index');
exports['fr_CA'] = require('./locales/fr_CA/index');
exports['ge'] = require('./locales/ge/index');
exports['id_ID'] = require('./locales/id_ID/index');
exports['it'] = require('./locales/it/index');
exports['ja'] = require('./locales/ja/index');
exports['ko'] = require('./locales/ko/index');
exports['nb_NO'] = require('./locales/nb_NO/index');
exports['nep'] = require('./locales/nep/index');
exports['nl'] = require('./locales/nl/index');
exports['pl'] = require('./locales/pl/index');
exports['pt_BR'] = require('./locales/pt_BR/index');
exports['ru'] = require('./locales/ru/index');
exports['sk'] = require('./locales/sk/index');
exports['sv'] = require('./locales/sv/index');
exports['tr'] = require('./locales/tr/index');
exports['uk'] = require('./locales/uk/index');
exports['vi'] = require('./locales/vi/index');
exports['zh_CN'] = require('./locales/zh_CN/index');
exports['zh_TW'] = require('./locales/zh_TW/index');
